### DISCUSSION TITLE
A static product info or launch website for the app

### DISCUSSION THREAD

**Discussion by aashishah** [avinashkranjan/Friday#91] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'd like to pitch the idea of a static website that showcases the app and all it's features, as well as other information, with an eventual download link.

**Describe the solution you'd like**
The website can be created using simple JS, HTML5 and CSS and hosted on Github.

**Describe alternatives you've considered**
We could first make a figma prototype/design and then turn it into an actual webpage(s).

**Comment by D-Ajay-Kumar** [comment]
Sounds interesting :D

**Comment by avinashkranjan** [comment]
Sounds Like a Great Idea.. Looking for people to work on it

**Reply by aashishah** [reply]
I'd like to work on this with people under GSSOC'21.

**Reply by avinashkranjan** [reply]
Can I get a wireframe or a prototype on which you are going to work?
